scripts/general_info.py

```diff
@@ -0,0 +1,144 @@
+import sys

```

Comment:
Better to convert this into a functional version, instead of scripting version. As when you import, it will be all loaded which seems a bad practice @Wolf-Legend

Reply:
Sure, will do that :) @vivonk